src/basic-syntax/static-and-const.md

```diff
@@ -38,18 +42,31 @@ fn main() {
 }
 ```
 
-As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and embedded code, and the variable lives through the entirety of the program execution.
+As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and 
+embedded code, and the variable lives through the entirety of the program execution.
 When a globally-scoped value does not have a reason to need object identity, `const` is generally preferred.
 
-Because `static` variables are accessible from any thread, they need to be guarded, for example by a [`Mutex`](https://doc.rust-lang.org/std/sync/struct.Mutex.html), or accessed using `unsafe` code.
-We will look at [mutating static data](../unsafe/mutable-static-variables.md) in the chapter on Unsafe Rust.
+Because `static` variables are accessible from any thread, they have more ownership constraints than data on the stack or the heap. 
+For instance, they cannot be safely mutated. We will look at [mutating static data](../unsafe/mutable-static-variables.md) in the chapter on Unsafe Rust..
 
 <details>
 
 * Mention that `const` behaves semantically similar to C++'s `constexpr`.
 * `static`, on the other hand, is much more similar to a `const` or mutable global variable in C++.
 * `static` provides object identity: an address in memory and state as required by types with interior mutability such as `Mutex<T>`.
 * It isn't super common that one would need a runtime evaluated constant, but it is helpful and safer than using a static.
+* `thread_local` data can be created with the macro `std::thread_local`.
+
+### Properties table: 
+
+| Property | Static | Constant |
+|---|---|---|
+| Allocated at runtime | Yes | No |
```

Comment:
What do you mean by this? Statics are generally allocated at compile-time, in the rodata section of the binary. Constants aren't 'allocated' at all, they are inlined.